test/WatchTestCases.template.js

```diff
@@ -141,6 +141,9 @@ const describeCases = config => {
 									if (!options.context) options.context = tempDirectory;
 									if (!options.entry) options.entry = "./index.js";
 									if (!options.target) options.target = "async-node";
+									if (!options.module) options.module = {};
+									if (!("unsafeCache" in options.module))
+										options.module.unsafeCache = false;
```

Comment:
Add a TODO here to remove that again `// TODO webpack 6 remove this after fixing unsafeCache`

Reply:
Or we wait for this to be fixed and remove it from the PR

Reply:
fixed by #14735